lib/hexpm/repository/packages.ex

```diff
@@ -9,6 +9,17 @@ defmodule Hexpm.Repository.Packages do
     Repo.one!(Package.count(repositories, filter))
   end
 
+  def diff(packages, nil), do: packages
+
+  def diff(packages, remove) when is_list(remove) do
+    names = remove |> Enum.map(& &1.name)
```

Comment:
```suggestion
    names = Enum.map(remove, & &1.name)
```

We usually try to avoid single pipes since they do not improve readability.